api-server-nest/src/utils/clone-attributes.ts

```diff
@@ -2,7 +2,13 @@ export default function cloneAttributes<T>(from: Partial<T>, to: T): void {
     Object.keys(from).forEach((key: string) => {
         const value = from[key]
 
-        if (Array.isArray(value)) {
+        if (value === undefined) {
+           return // skip
+        }
+
+        if (value === null) {
+            to[key] = null
+        } else if (Array.isArray(value)) {
```

Comment:
Is there a particular reason to write this util ourselves vs use one from Underscore/Lodash?

Reply:
I got scared away from lodash. I've heard folks at work complain that they don't patch security vulns and have been hostile to attempts to patch major problems, I don't know the details but comes from people I trust with a record of contributing to open source projects (though now I'm not sure if it was lodash or underscore) :shrug:

In any event it's a fair point, not good to reinvent the wheel. We should probably look into that. The good news is that by having it in our little wrapper like this we could just swap it out for lodash under the hood at some point